scripts/generate-federate-match.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+shopt -s inherit_errexit
+
+! [ -x "$(command -v jq)" ] && echo 'jq not installed, the script requires it.' && exit 1
+! [ -x "$(command -v mimirtool)" ] && echo 'mimirtool not installed, the script requires it.' && exit 1
+! [ -x "$(command -v realpath)" ] && echo 'realpath not installed, the script requires it.' && exit 1
+! [ -x "$(command -v sort)" ] && echo 'sort not installed, the script requires it.' && exit 1
+! [ -x "$(command -v uniq)" ] && echo 'uniq not installed, the script requires it.' && exit 1
+! [ -x "$(command -v yq)" ] && echo 'yq not installed, the script requires it.' && exit 1
+
+function log() {
+    echo "${1:-}" >&2
```

Comment:
Nit: this way subsequent args will not be silently ignored:
```suggestion
    echo "$@" >&2
```